bugfix button component

Olena-Horobets opened the pull request:
Добавила стили для мобилки для btn--contacts - ширина фиксированная
Прописала в стили размер иконки стрелки, из разметки его можно убрать

Olena-Horobets committed:
bugfix button component

Добавила стили для мобилки для btn--contacts - ширина фиксированная
Прописала в стили размер иконки стрелки, из разметки его можно убрать

Valeriy-Fedulov commented on src/sass/components/_buttons.scss:
```diff
@@ -96,17 +109,3 @@
-
-  @include respond-above($desktop) {
-    font-size: 16px;
-    line-height: 1.31;
```
На desktop мне нужно чтобы менялся размер шрифта!!!

Valeriy-Fedulov replied:
Плиз оставьте мне 16px

Olena-Horobets replied:
он остался, я переместила по порядку секций

Olena-Horobets replied:
.btn--about {
  font-size: 14px;
  line-height: 1.29;

  @include respond-above($desktop) {
    font-size: 16px;
    line-height: 1.31;
  }
}
 всё есть

Olena-Horobets replied:
header
hero
about
locations

Olena-Horobets committed:
update bugfix

1. class btn--contacts chaged to btn--locations:
2. mobile styles for this class fixed;
